### PR TITLE
Add force-chinese-ime

### PR DESCRIPTION
**Pull request by u3l6**

### CODE REVIEW

File: mods/force-chinese-ime.wh.cpp [hunk 1]
```diff
@@ -0,0 +1,109 @@
+// ==WindhawkMod==
+// @id              force-chinese-ime
+// @name            Force Chinese IME Mode
+// @description     Forces Microsoft Pinyin IME to stay in Chinese mode
+// @version         1.0
+// @author          u3l6
+// @github          https://github.com/u3l6
+// @twitter         https://x.com/u_3l6
```
Comment (m417z) [inline mods/force-chinese-ime.wh.cpp:8]: Please send me (https://x.com/m417z) a direct message with the following content:

> I attest that I'm the sole owner of both this X account (https://x.com/u_3l6) and the following GitHub account: https://github.com/u3l6

File: mods/force-chinese-ime.wh.cpp [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        (void**)&ImmSetConversionStatus_Original // Pass original function pointer here
+    );
+
+    Wh_Log(L"Force Chinese IME Mode mod initialized.");
+}
+
+// Module cleanup
+void ModUninit() {
+    // Remove the hook
+    Wh_RemoveFunctionHook((void*)ImmSetConversionStatus);
```
Comment (m417z) [inline mods/force-chinese-ime.wh.cpp]: That's unnecessary. At this point, the hook is already removed, see [Mod lifetime](https://github.com/ramensoftware/windhawk/wiki/Mod-lifetime).

Comment (intoop) [reply mods/force-chinese-ime.wh.cpp]: OK, I removed this part

**Comment by m417z** [comment]
Congratulations on your first submission :)

**Comment by u3l6** [comment]
Thank you for making the mod :)

**Pull request merged**